### PR TITLE
Append Windows-CoreCLR installation instructions.

### PR DESCRIPTION
**Pull request by alxest**
I was stuck here and thought it could be helpful for others like me.
Added just two lines.

### CODE REVIEW

File: Documentation/Getting-Started-For-Windows.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ may be replaced in all commands below with the directory of your choice.
 * Clone and build CoreCLR:
     ```
     $ git clone https://github.com/dotnet/coreclr
+    $ git clone https://github.com/dotnet/corefx
```
Comment (JosephTremoulet) [inline Documentation/Getting-Started-For-Windows.md]: This didn't used to be necessary; coreclr build would pull in pre-built corefx bits with nuget.
@mmitche / @adiaaida / @richardlford, has something changed that makes it necessary to build corefx locally?  Do our CI jobs build corefx locally now, or still use nuget?
@alxest, what was the first error you got when you omitted this line?

Comment (adiaaida) [inline Documentation/Getting-Started-For-Windows.md]: No, not as far as I know. I remember there being some issue with nuget last week (@mmitche would have more info), but in the lab we do not build corefx locally, so it should not be necessary.

Comment (mmitche) [inline Documentation/Getting-Started-For-Windows.md]: AFAIK there is no requirement to build corefx when building coreclr.

Nuget saw issues when there was an Azure network issue.

Comment (richardlford) [inline Documentation/Getting-Started-For-Windows.md]: Building coreclr just requires running their build.cmd. corefx is not required.

Comment (russellhadley) [inline Documentation/Getting-Started-For-Windows.md]: @adiaaida @alxest did we get to the bottom of this?  I'm wondering why @alxest needed to add corefx.

Comment (adiaaida) [inline Documentation/Getting-Started-For-Windows.md]: I don't know. I don't know what issues he was seeing when he didn't clone corefx. In the lab, and locally for me, this was not necessary.

**Comment by alxest** [comment]
I removed corefx repository then tried it again, and it was successful.
Sorry for the confusion.

**Pull request closed without merging**